main.js

```diff
@@ -1,73 +1,205 @@
+      'w-100',
+      'text-center',
+      'content-box',
+      'content',
+      'pt-2',
+    );
+    this.contactInfos.setAttribute('id', 'contact');
+    this.contactInfos.classList.add('contact');
+    this.contactInfos.innerHTML = `<div class="flex-center-column contactBox w-75 p-1">
+                                       <div class="pb-2 w-100 contact-title flex-center-column">
+                                         <h3 class="fw-bold fs-4 w-100 flex-center-column position-relative">Contact informarion</h3>
+                                       </div>
+                                       <p>Do you have any question or you just want to say <span class="fw-bolder">'Hello!'</span><p/>
+                                       <p>You can reach out to us!</p>
+                                       <ul class="list-group pt-2 text-start contact-list-container">
+                                         <li class="list-group-item">Our e-mail:<span class="fw-bold"> mail@gmail.com </span></li>
+                                         <li class="list-group-item">Our Phone number:<span class="fw-bold"> 0043729136280 </span></li>
+                                         <li class="list-group-item">Our address:<span class="fw-bold"> streetname 22, 84503 city, country</span></li>
+                                       </ul>
+                                     </div>`;
+    this.footer = document.createElement('footer');
+    this.footer.setAttribute('id', 'footer');
+    this.footer.classList.add(
+      'footer',
+      'position-fixed',
+      'bottom-0',
+      'w-100',
+    );
+    this.footer.innerHTML = `<div class="copyright py-3 text-start me-3">
+                                 <p class="h6 text-white">Copyright </p>
+                               </div>`;
+    this.container.append(this.header, this.time, this.main, this.footer);
+    this.main.append(this.bookList, this.NewBookForm, this.contactInfos);
+  }
 
-  // Add Book to UI
-  UI.addBookToList(book);
+  update() {
+    UI.displayBook();
+    const nav = new Navigation();
+    nav.init();
+  }
+}
 
-  // Add book to storage
-  Store.addBook(book);
+const layout = new Layout();
+layout.update();
 
-  // Clear fields
-  UI.clearFields();
+document.querySelector('.listDisplay').addEventListener('click', (e) => {
+  const isButton = e.target.nodeName === 'BUTTON';
+  if (!isButton) {
+    return;
+  }
+  UI.deleteBook(e.target.parentElement.id);
+  Store.removeBook(e.target.parentElement.id);
 });
 
-// Delete a book
-document.querySelector('#book-list').addEventListener('click', (e) => {
-  // remove book from UI
-  UI.deleteBook(e.target);
+// Imlplement responsive menu
+
+const menu = document.querySelector('.header');
 
-  // remove book from broswer storage according to author
-  Store.removeBook(e.target.parentElement.previousElementSibling.textContent);
+document.getElementById('humburger').onclick = () => {
+  menu.classList.add('change-menu');
+  // alert('hello')
```

Comment:
- [ ] Kindly, remember not to leave commented code 🍡, always you should maintain best practices in JavaScript. Please, remove commented code 🔢.

Reply:
@fherrerao 
Removed commented code at this [commit](https://github.com/Lucash2022/Awesome_Book/commit/7d237603c0d4e20b0888fbbd42b53f3034bfca1a)